docs/crud/stream/schema.md

```diff
@@ -0,0 +1,109 @@
+# CRUD Streams
+
+CRUD operations create, read, update and delete streams.
+
+[`bydbctl`](../../clients.md#command-line) is the command line tool in examples.
+
+Stream intends to store streaming data, for example, traces or logs.
+## Create operation
+
+Create operation adds a new stream to the database's metadata registry repository. If the stream does not currently exist, create operation will create the schema.
+
+### Examples of creating
+
+A stream belongs to a unique group. We should create such a group with a catalog `CATALOG_STREAM`
+before creating a stream.
+
+```shell
+$ bydbctl group create -f - <<EOF
+metadata:
+  name: default
+catalog: CATALOG_STREAM
+resource_opts:
+  shard_num: 2
+  block_interval:
+    unit: UNIT_HOUR
+    num: 2
+  segment_interval:
+    unit: UNIT_DAY
+    num: 1
```

Comment:
A small discussion, would we support block_size and segment_size in the future or now?
In a typical use case, size-oriented policy is widely used for storage/disk operation.

Reply:
Of course, in those cases, we should discuss TTL policy as well. Because they are relative logically.

Reply:
We will set the maximum number of bytes of storage blocks to retain in a shard. Some algorithms are based on time, and can not work with fixed-size segments/blocks. Theoretically, The oldest blocks will be removed first. 

If TTL conflicts with the max size, they could work separately. For instance, a block that doesn't reach its time end can be removed by the size limitation controller. This strategy is a canonical way and opted for by several TSDBs.

Reply:
The typical setup could be a threshold for the minimal size of the block, which is expected to be pre-allocated by the operation team.